src/main/java/org/threadly/concurrent/NoThreadScheduler.java

```diff
@@ -53,10 +69,15 @@ public NoThreadScheduler() {
    * Abstract call to get the value the scheduler should use to represent the current time.  This 
    * can be overridden if someone wanted to artificially change the time.
    * 
+   * @param accurate If {@code true} then time estimates are not acceptable
    * @return current time in milliseconds
    */
-  protected long nowInMillis() {
-    return Clock.accurateForwardProgressingMillis();
+  protected long nowInMillis(boolean accurate) {
```

Comment:
if kept might be worth just making a static also, I think the compiler can optimize it more then.